src/components/currency/Currency.js

```diff
@@ -7,24 +7,20 @@ import {NumberComponent} from '../number/Number';
 export class CurrencyComponent extends NumberComponent {
   constructor(component, options, data) {
     super(component, options, data);
-
     this.currency = this.component.currency || 'USD';
-    this.decimalLimit = this.component.decimalLimit || 2;
-
+    this.decimalLimit = options.decimalLimit || this.component.decimalLimit || 2;
```

Comment:
Is there a reason we are adding an options.decimalLimit override? I don't think this was a requirement.

Reply:
I think there is a Requirement.